source/theory.rst

```diff
@@ -75,6 +80,11 @@ The buoyancy term :math:`F_{B}(t)` depends on the hydrostatic stiffness :math:`K
 
 Numerical Methods
 ------------------
+.. Note:: 
+	Adam:
+	This portion is a big confusing. I can follow the derivation and content within each subsection, but several steps are skipped between the subsections. i.e. between 'Convolution Integral formulation' to 'State Space' and 'Calculation of Kr...' to ' Realization Theory' Maybe it is desirable and necessary to leave the full explanation to other papers, but I can not get very far with this short explanation. I don't currently have the knowledge on the topic to identify what is missing and needs to be included to complete the section.
+	What is the difference between B_r, B_d, B_i,j? It is not well documented
+
```

Comment:
I agree, this section goes into great detail on some topics, but skims or fails to address others... I think updating the Theory should be its own card on the doc project board.

Reply:
@kmruehl Good idea, I added a distinct card for this on the documentation project